gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -241,10 +246,14 @@ private void benchmarkWrite(
                         output.write(writeBuffer);
                         fileBytesWrite += writeSize;
                         writeCallBytes.accept(writeSize);
+
+                        writeFileTime.add(System.nanoTime() - writeCallStart);
                         writeCallTimeNs.accept(System.nanoTime() - writeCallStart);
                       } while (fileBytesWrite < totalSize);
 
                       writeFileBytes.accept(fileBytesWrite);
+
+                      writeFileTime.add(System.nanoTime() - writeStart);
```

Comment:
What is the difference b/w this and writeFileTimeNs?

Reply:
From writeFileTimeNs, we cannot calculate p50, p95 etc as from object of class LongSummaryStatistics, we cant get list of values stored imo.